cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -410,11 +410,8 @@ def _add_iam_lambda_role(self):
                                 actions=s3_policy_actions,
                                 effect=Effect.ALLOW,
                                 resources=[
-                                    self.format_arn(service="s3", resource=self._bucket.name),
-                                    self.format_arn(
-                                        service="s3",
-                                        resource="{0}/{1}/*".format(self._bucket.name, self._bucket.artifact_directory),
-                                    ),
+                                    f"arn:aws:s3:::{self._bucket.name}",
```

Comment:
The "aws" partition must be changed to "aws-cn" when we're in china. I'd keep format_arn because it automatically manages it.

Reply:
Done